benchmarks/bench_lof.py

```diff
@@ -21,17 +21,20 @@
 np.random.seed(2)
 
 # datasets available: ['http', 'smtp', 'SA', 'SF', 'shuttle', 'forestcover']
-datasets = ['http', 'smtp', 'SA', 'SF', 'shuttle', 'forestcover']
+datasets = ['shuttle']
```

Comment:
Because it is computationally very heavy to run on all these datasets. shuttle is the lightest one.